custom/sensors/CANTalonEncoder.java

```diff
@@ -21,129 +23,129 @@ public CANTalonEncoder(String name, CANTalon talon, boolean reverseDirection, do
 	public CANTalonEncoder(String name, CANTalon talon, boolean reverseDirection) {
 		this(name, talon, reverseDirection, 1.0);
 	}
-	
+
 	public CANTalonEncoder(String name, CANTalon talon, double distancePerPulse) {
 		this(name, talon, false, distancePerPulse);
 	}
-	
+
 	public CANTalonEncoder(String name, CANTalon talon) {
 		this(name, talon, false);
 	}
-	
+
 	public CANTalonEncoder(CANTalon talon, boolean reverseDirection, double distancePerPulse) {
 		this("CANTalonEncoder", talon, reverseDirection, distancePerPulse);
 	}
-	
+
 	public CANTalonEncoder(CANTalon talon, double distancePerPulse) {
 		this("CANTalonEncoder", talon, distancePerPulse);
 	}
-	
+
 	public CANTalonEncoder(CANTalon talon) {
 		this("CANTalonEncoder", talon);
 	}
-	
+
 	@Override
 	public void setPIDSourceType(PIDSourceType pidSource) {
 		this.pidSource = pidSource;
 	}
-	
+
 	@Override
 	public PIDSourceType getPIDSourceType() {
 		return pidSource;
 	}
-	
+
 	@Override
 	public double pidGet() {
 		if (pidSource == PIDSourceType.kDisplacement) {
 			return getDistance();
 		}
 		return getRate();
 	}
-	
+
 	@Override
 	public int get() {
-		return talon.getEncPosition();
+		return (int) talon.getPosition();
 	}
-	
+
 	@Override
 	public double getDistance() {
 		if (reverseDirection) {
-			return distancePerPulse * talon.getEncPosition() * -1.0;
+			return distancePerPulse * talon.getPosition() * -1.0;
 		} else {
-			return distancePerPulse * talon.getEncPosition();
+			return distancePerPulse * talon.getPosition();
 		}
 	}
-	
+
 	@Override
 	public boolean getDirection() {
-		return !reverseDirection == (talon.getEncVelocity() >= 0);
+		return !reverseDirection == (talon.getSpeed() >= 0);
 	}
-	
+
 	@Override
 	public boolean getStopped() {
 		return Util.isZero(getRate());
 	}
-	
+
 	@Override
 	public double getRate() {
 		if (reverseDirection) {
-			return talon.getEncVelocity() * -10.0 * distancePerPulse;
-			// getEncVelocity must be converted from ticks to 100ms to ticks per second, so *10.
+			return talon.getSpeed() * -10.0 * distancePerPulse;
+			// getSpeed must be converted from ticks to 100ms to ticks per second, so *10.
```

Comment:
Based on code from here: http://first.wpi.edu/FRC/roborio/release/docs/cpp/classCANTalon.html#a36e486b58bc5ce1b218d36468900bb1b, `The speed units will be in the sensor's native ticks per 100ms.`, so that is still correct.
Also, the derivative is taken on the Talon (similar to how the Teensys do it), so it is more precise and we do not have to care.